src/oss/confluent.sh

```diff
@@ -1355,13 +1390,26 @@ These are the available commands:
 
     config      Configure a connector.
 
+EOF
+for ((i = 0; i < ${#dynamic_cli_commands[@]}; ++i)); do
+local command=${dynamic_cli_commands[$i]}
+local desc=${dynamic_cli_command_descs[$i]}
+if dynamic_cli_command_exists "${command}" && dynamic_cli_file_exists "${command}"; then
```

Comment:
This behavior here is related to my comments about COS vs CP enterprise. 

As I imply above, with respect to the list of commands we should be a bit static. We need two categories of commands available (for instance listing them when you run `confluent help`). Ubiquitous commands and enterprise-only commands. 

Now if security-plugins is something that might or might not be available with enterprise based on whether a user has purchased this package, I believe the way to go w.r.t visibility is not to hide the command. Instead we should list the command and if the user tries to run it, and security-plugins are not there, it should fail gracefully with a message that says: "ACL functionality is not available with your Confluent installation ... blah instructions about how to purchase, etc". 

The reason I am specific with that is that, if let's say a user deletes by accident the ./bin/acl-cli, Confluent CLI should warn instead of just not showing the command. If it's an enterprise only feature, then yes, COS `confluent help` will not have it there at all. 

The criterion based on which we'll decide whether the deployment is COS or CP through the CLI is TBD. (I can get back to you on that).